modules/bs.string/data/bs.string/function/find/normal/skip.mcfunction

```diff
@@ -0,0 +1,22 @@
+# ------------------------------------------------------------------------------------------------------------
+# Copyright (c) 2025 Gunivers
+#
+# This file is part of the Bookshelf project (https://github.com/mcbookshelf/Bookshelf).
+#
+# This source code is subject to the terms of the Mozilla Public License, v. 2.0.
+# If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Conditions:
+# - You may use this file in compliance with the MPL v2.0
+# - Any modifications must be documented and disclosed under the same license
+#
+# For more details, refer to the MPL v2.0.
+# ------------------------------------------------------------------------------------------------------------
+
+$execute store result storage bs:ctx x int 1 run scoreboard players add #i bs.ctx $(z)
```

Comment:
As said in find, it's better to only work with score and only copy to out storage when something is found